packages/jbrowse-web/src/rootModel.ts

```diff
@@ -276,6 +276,8 @@ export function createTestSession(snapshot = {}) {
     ...snapshot,
   })
   root.setHistory(UndoManager.create({}, { targetStore: root.session }))
+  // @ts-ignore
+  root.session.views.map(view => view.setWidth(800))
```

Comment:
why are we manually setting the width here, can't we wait for the overall view to set it?

Reply:
thjat particular line is for creating test snapshots, and the jest-dom does not do this for us